Problem1.py.txt

```diff
@@ -0,0 +1,45 @@
+# Time Complexity : O(n^1/2) to initialize array??, O(1) put, O(1) get, O(1) remove
```

Comment:
Have it as null itself and initialize in put() if null is still there. This avoids that initialization issue.